src/new-bot/commands/expertise/update.js

```diff
@@ -36,16 +39,143 @@ function updateExpertise({userName, expertise, newValues}) {
   });
 }
 
-const updateMissingHandler = createMatcher({match: 'missing'}, (_, {user}) => {
-  return `> update missing for ${user.name} (coming soon)`;
-});
+function updateExpertiseDialog({
```

Comment:
All it does is get some data from the DB and display a series of dialogs. Unfortunately, those dialogs are a long chain of questions and message handlers, with conditional flow. Do you have any suggestions on how to split this up?